openbb_platform/core/openbb_core/provider/utils/errors.py

```diff
@@ -12,3 +12,15 @@ def __init__(
         """Initialize the exception."""
         self.message = message
         super().__init__(self.message)
+
+
+class UnauthorizedError(OpenBBError):
+    """Exception raised for an unauthorized request response."""
+
+    def __init__(
+        self,
+        message: str = "Unauthorized request. Please check your credentials and subscription access from the provider.",
```

Comment:
How would you further clarify the default message? This tells you it is a 'provider', and in most cases the default message will not be displayed because the error message from the provider is being directly relayed instead.

> Unauthorized request. Please check your credentials and subscription access from the provider.